osu.Game/Screens/OnlinePlay/Multiplayer/Match/MatchStartControl.cs

```diff
@@ -251,6 +254,28 @@ private void updateState()
             });
         }
 
+        public bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
+        {
+            if (!readyButton.Enabled.Value)
+            {
+                return false;
+            }
+
+            switch (e.Action)
+            {
+                case GlobalAction.MultiplayerReady:
+                    onReadyButtonClick();
```

Comment:
```suggestion
                    readyButton.TriggerClick();
```
is preferred as that should play out all of the related button animations that play when the button is clicked using the mouse. It's a UX thing. Same for the other hotkey.

Reply:
Ah, makes sense. By doing things that way it also looks like the checks I previously did with `operationInProgress.Value` and `!readyButton.Enabled.Value` are now useless, so I got rid of those.